### ISSUE TITLE
pts/rav1e-1.3.1 appears to be broken

### ISSUE THREAD

**Issue by nergdron** [phoronix-test-suite/test-profiles#175]
I'm attempting to run the latest rav1e benchmark, but in the install phase it fails with the following error message:

`error: no override and no default toolchain set`

not exactly the most helpful. I'm on Ubuntu 20.04, running `rustc 1.47.0 (18bf6b4f0 2020-10-07)`. there's no obvious compile failures, and there is what appears to be a valid binary in `~/.phoronix-test-suite/installed-tests/pts/rav1e-1.3.1/rav1e`.

**Comment by michaellarabel** [comment]
It's a known bug with rav1e on Ubuntu 20.04 LTS' default Rust components. Presumably will be fixed with the next rav1 release.

**Comment by nergdron** [comment]
further investigation trying to manually build reveals rust build errors:

```
error: failed to run custom build command for `rav1e v0.3.0 (/home/tessa/.phoronix-test-suite/installed-tests/pts/rav1e-1.3.1/rav1e-0.3.0)`

Caused by:
  process didn't exit successfully: `/home/tessa/.phoronix-test-suite/installed-tests/pts/rav1e-1.3.1/rav1e-0.3.0/target/debug/build/rav1e-b1fe64d957862d88/build-script-build` (exit code: 101)
  --- stdout
  cargo:rustc-cfg=nasm_x86_64

  --- stderr
  thread 'main' panicked at 'Invalid version component: ParseIntError { kind: InvalidDigit }', /home/tessa/.cargo/registry/src/github.com-1ecc6299db9ec823/nasm-rs-0.1.8/src/lib.rs:325:40
```

and some searching reveals this upstream issue:

https://github.com/xiph/rav1e/issues/2482

so it looks like this test should be updated to rav1e version >= 0.3.4 and it should solve this problem.

**Comment by michaellarabel** [comment]
pts/rav1e-1.4.0 is now available that updates against rav1e 0.4 Alpha upstream and should fix this problem. Thanks and sorry for the delay.
